client/src/api/datasource/ApiClient.ts

```diff
@@ -5,11 +5,36 @@ import { SigMFMetadata } from '@/Utils/sigmfMetadata';
 
 export class ApiClient implements DataSourceClient {
   async list(): Promise<DataSource[]> {
-    const response = await axios.get('/api/datasources');
+    // TODO: Why doesn't this get routed correctly when called from the test
```

Comment:
What do you mean?

Reply:
I think Tobe had playwright config selection issues.